pipeline/util/__init__.py

```diff
@@ -29,10 +29,9 @@ def hex_to_python_object(hex: str) -> Any:
 
 
 def python_object_to_name(obj: Any) -> Optional[str]:
-    try:
-        name = obj.__name__
-    except Exception:
-        name = None
+    # Consider limiting the size of the name in future releases
+    name = getattr(obj, "__name__", str(obj))
```

Comment:
Is the string representation of an object also a good name?

Reply:
Ahhhh so this function is poorly named, the `obj` here is actually the type of the variable passed used from a functions annotations

Reply:
Types are also objects so the question is relevant, but as discussed this is understood to not be perfect but better (supports more types) than what it replaces.